compat-28.el

```diff
@@ -392,6 +392,39 @@ not a list, return a one-element list containing OBJECT."
       object
     (list object)))
 
+(compat-defun get-buffer-create (name &optional inhibit-buffer-hooks)
+  "Return the buffer specified by BUFFER-OR-NAME, creating a new one if needed.
```

Comment:
The doc string can be shorter here. It is sufficient to write only "This extended function implements the INHIBIT-BUFFER-HOOKS argument.", such that it is obvious that the function only adds new functionality over the existing `get-buffer-create`. See the other doc strings of `:extended t` functions for reference.